gctf/__init__.py

```diff
@@ -59,9 +59,6 @@ def getEnviron(cls):
 
     @classmethod
     def defineBinaries(cls, env):
-        env.addPackage('gctf', version='0.50',
-                       tar='Gctf_v0.50.tgz')
-
         env.addPackage('gctf', version='1.06',
```

Comment:
Dont' we need also a binary for version 1.18?

Reply:
it's already there in the code (devel) so it's not shown in this pr